src/libraries/Microsoft.PowerFx.Interpreter/ParsedExpression.cs

```diff
@@ -101,6 +104,7 @@ internal class ParsedExpression : IExpressionEvaluator
         internal ReadOnlySymbolValues _globals;
         internal ReadOnlySymbolTable _allSymbols;
         internal ReadOnlySymbolTable _parameterSymbolTable;
+        internal PowerFxConfig _config;

```

Comment:
>_config [](http://example.com/codeflow?start=31&length=7)

we should be able to avoid adding  this, and get via IRuntimeConfig instead.

Reply:
why is this a problem?